probe_scraper/scrapers/git_scraper.py

```diff
@@ -122,8 +122,8 @@ def retrieve_files(
     results = defaultdict(list)
     timestamps = dict()
     base_path = cache_dir / repo_info.name
-    repo_name = repo_info.url.rstrip("/").split("/")[-1]
-    repo_path = cache_dir / f"{repo_name}.git"
+    org_name, repo_name = repo_info.url.rstrip("/").split("/")[-2:]
+    repo_path = cache_dir / org_name / f"{repo_name}.git"
```

Comment:
fixes conflict between bergamot's url `https://github.com/mozilla-extensions/firefox-translations` and firefox-translations' url `https://github.com/mozilla/firefox-translations`